config/config.yaml

```diff
@@ -0,0 +1,52 @@
+/**:
+  # Global Parameters
+  ros__parameters:
+    use_sim_time: true
+    tf_timeout_threshold: 0.1
```

Comment:
don't think it is a good idea to place it here

Reply:
Where would you put it? I think it's clearest to indicate whether all nodes should use `use_sim_time` or not, and the `tf_timeout_threshold` would be the same for everyone.

Reply:
My comment was only about `tf_timeout_threshold`, sorry it was unclear. Not sure about that would be the same for every behavior. We used to set a different value for `go_to`  or `follow_path`. Even if you add new behaviors as `follow_reference` or `point_gimbal`, they are going to also use the same value. It seems a bit greedy to me. I would place it at each behavior separately.

Reply:
I set it because the default value is very low, and computers with limited resources encounter errors. I think that by default, a generic value can be set for all nodes so that if the tf fails, the user only needs to increase that value and doesn't have to adjust this parameter for each node individually. Typically, the controller and behaviors encounter failures, and by setting the same value for all, it works because the buffer is updated the same way for everyone.

Reply:
I agree, it is good to have it set, but I disagree in the placement. Indeed, I believe we should have chosen different names for each parameter